### DISCUSSION TITLE
Incomplete plot path modifier doesn't show an error

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#541] [Q&A]
When I've partially typed `.@derivative` on my plot path, the text in the field should be red because the path is invalid. However it looks valid:

<img width="341" alt="Screen Shot 2022-01-19 at 1 50 56 PM" src="https://user-images.githubusercontent.com/14237/150219707-33ed3f8a-fe40-45e3-a196-49d835da0649.png">

**Comment by banisadr** [comment]
This appears to still be an issue 😭 - filing ticket

**Reply by defunctzombie** [reply]
👍 I definitely tested it before closing this ... so I look forward to your reproduction report.

**Reply by banisadr** [reply]
I also tested it... I'm confused. Filed FG-7987, currently seeing the issue on my screen as I type this 🤷 
<img width="1840" alt="image" src="https://github.com/foxglove/community/assets/12058745/b58d825e-27e7-4d88-9ea3-8385fd89d89c">

**Reply by defunctzombie** [reply]
<img width="497" alt="image" src="https://github.com/foxglove/community/assets/84792/02cb8d51-d0e7-4b6e-a29f-f8490a4b1439">

However, adding a `.` after the `.x` makes the error go away and instead I am left with a valid input and data plots. So that should provide a hint to where things went went.

<img width="447" alt="image" src="https://github.com/foxglove/community/assets/84792/e60e0acd-5710-4b51-9631-a46d4faf1044">

**Reply by banisadr** [reply]
Isn't that what @jtbandes original item is complaining about? 
> When I've partially typed .@derivative on my plot path, the text in the field should be red because the path is invalid.

The math modifier syntax is `.@` so `/imu.linear_accel.x@abs` is invalid syntax & correctly highlights as red in the app. 
<img width="629" alt="image" src="https://github.com/foxglove/community/assets/12058745/19697b94-2011-4e77-be03-7abd56e84341">

The problem is valid syntax with an invalid math modifier (`/imu.linear_accel.x.@absu`) is accepted but shouldn't be.

**Comment by banisadr** [comment]
This has been resolved 🙌
